src/style.css

```diff
@@ -0,0 +1,141 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  font-family: inherit;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__logo {
+  background-repeat: no-repeat;
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.card__text {
+  font-family: inherit;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  width: 166px;
+  height: 36px;
```

Comment:
don't hardcode with width and height
use paddings for card instead of widths and don't use height for text